src/osa/scripts/datasequence.py

```diff
@@ -189,6 +231,10 @@ def dl1ab(run_str: str) -> int:
     input_dl1_datafile = Path(options.directory) / f"dl1_LST-1.Run{run_str}.h5"
     # DL1b output file to be stored in the dl1ab subdirectory
     output_dl1_datafile = dl1ab_subdirectory / f"dl1_LST-1.Run{run_str}.h5"
+    night_dir = date_to_dir(options.date)
+    calib_prod_id = cfg.get("LST1", "CALIB_PROD_ID")
```

Comment:
maybe this calib prod id could be simply the lstchain major version? v0.10, v0.11, etc